mars-rover/mars_rover.rb

```diff
@@ -1,3 +1,151 @@
+# class Dimensions is to define the maximum size of the map (N, E, S, W)
+class Dimensions
+
+   def initialize (n,e,s,w)
+     $max_n = n
+	 $max_e = e
+	 $max_s = s
+	 $max_w = w
+   end
+   
+   def get_dimensions
+      return [$max_n,$max_e,$max_s,$max_w]
+   end
+ end
+
+# class Obstacle is to define a position of an obstacle on the map
+class Obstacle
+
+  def initialize (x,y)
+   $obstacle_pos_x = x
+   $obstacle_pos_y = y
+  end
+
+  def get_obstacle_position
```

Comment:
The name doesn't have to include the class name, you'll be talking to an object anyway, so 'obstacle.position' is probably more natural than 'obstacle.get_obstacle_position'.